tensorflow/contrib/image/python/ops/single_image_random_dot_stereograms.py

```diff
@@ -34,8 +34,6 @@ def single_image_random_dot_stereograms(
     normalize=None, normalize_max=None,
     normalize_min=None,
     border_level=None,
-    number_colors=None,
```

Comment:
This was an error, I don't understand how it compiled and tested properly.  Maybe the older version of the PY file was held by Python Notebook.  It is fixed in the next commit.

Sorry, good catch.